MIGRATION.md

```diff
@@ -1,7 +1,7 @@
 #  Migration Notes
 
 ## Upcoming release
-- `telephoneNumber` property of `PrefilledShopperInformation` has been deprecated. Use to `phoneNumber` property if needed. 
```

Comment:
Thinking whether we should move it to the bottom of the list to keep the order consistent or whether we should sort the list the other way around so the newest release is always on the top 🤔 

What do you think? Also maybe good to have this discussion with @descorp  & @erenbesel :)

Reply:
87d6b7317958c614da3a726848fb9dbd25ff4f8c